assets/pages/showcase/showcase.scss

```diff
@@ -5,16 +5,9 @@
 // ----- Shared Components ----- //
 @import '~components/footer/footer';
 @import '~components/headers/simpleHeader/simpleHeader';
-
 @import '~components/svgs/svg';
-
 @import '~components/headingBlock/headingBlock';
 @import '~components/leftMarginSection/leftMarginSection';
```

Comment:
Not for this PR but I wonder if we should import this from `productPageContentBlock.js`, as it is a dependency? I guess ideally we'd want it to be in `leftMarginSection.js` itself though…

Reply:
Yeah, the components themselves should def import their own styles rather than the page that composes them together.